src/Util/ValueHelper.php

```diff
@@ -50,8 +50,8 @@ function compareArguments(array $argumentsA, array $argumentsB): bool
     }
 
     /**
-     * @param $valueA
-     * @param $valueB
+     * @param ValueNodeInterface $valueA
+     * @param ValueNodeInterface $valueB
      * @return bool
      */
     public function compareValues($valueA, $valueB): bool
```

Comment:
Could these even be type-hinted? @Jalle19

Reply:
Yes, I had to rebase on master and this change got lost